### PR TITLE
Fix `functions/satisfies` error when parsing JSLite code

### PR DESCRIPTION
**Pull request by mandx**

### CODE REVIEW

File: packages/core/src/parsers/jsx.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ import { stripNewlinesInStrings } from '../helpers/replace-new-lines-in-strings'
 
 const jsxPlugin = require('@babel/plugin-syntax-jsx');
 const tsPreset = require('@babel/preset-typescript');
+const _semver = require('semver');
```
Comment (steve8708) [inline packages/core/src/parsers/jsx.ts:15]: can you explain more here? not sure I understand the problem or the solution

Comment (steve8708) [reply packages/core/src/parsers/jsx.ts:15]: if you just need to pull it in maybe just do `require('semver')` ?

Comment (steve8708) [reply packages/core/src/parsers/jsx.ts:15]: seems the fiddle is unaffected by this, maybe this should be in the builder webapp instead? maybe just the require and a comment. I'll take a look at

Comment (steve8708) [reply packages/core/src/parsers/jsx.ts:15]: I'm unable to reproduce locally so seems this is a production only issue, how confident are you that this is the fix? and can it live in the Builder.io webapp instead if that's the only place this issue exists?

Comment (mandx) [reply packages/core/src/parsers/jsx.ts:15]: I was able to repro locally... I'll try again, and also try adding a `require('semver')` in the webapp instead and check if the error goes away.

**Review by steve8708 — APPROVED** [review]
perfect, thanks 🙏

**Pull request merged**